integrationExamples/gpt/esp_example.html

```diff
@@ -0,0 +1,210 @@
+                        syncDelay: 3000,
+                        userIds: [{
+                            name: 'id5Id',
+                            params: {
+                                partner: 173,            // change to the Partner Number you received from ID5
+                                abTesting: {             // optional
+                                    enabled: true,         // false by default
+                                    controlGroupPct: 0.1   // valid values are 0.0 - 1.0 (inclusive)
+                                }
+                            },
+                            storage: {
+                                type: 'html5',           // "html5" is the required storage type
+                                name: 'id5id',           // "id5id" is the required storage name
+                                expires: 90,             // storage lasts for 90 days
+                                refreshInSeconds: 8 * 3600 // refresh ID every 8 hours to ensure it's fresh
+                            }
+                        },{
+                            name: "sharedId",
+                            storage: {
+                                type: "cookie",
+                                name: `_pubcid`,         // create a cookie with this name
+                                expires: 365             // expires in 1 years
+                            }
+                        },{
+                            name: "criteo",
+                        },{
+                            name: "pubCommonId",
+                            storage: {
+                                type: "cookie",
+                                name: "_pubcid",         // create a cookie with this name
+                                expires: 365             // expires in 1 years
+                            }
+                        },{
+                            name: 'hadronId',
+                            storage: {
+                                name: 'hadronId',
+                                type: 'html5'
+                            }
+                        }],
+                        auctionDelay: 10000
+                    }
+                });
+                pbjs.requestBids({
+                    bidsBackHandler: initAdserver,
+                    timeout: PREBID_TIMEOUT
+                });
+            });
+
+
+            
+
+            function initAdserver() {
+                if (pbjs.initAdserverSet) return;
+                pbjs.initAdserverSet = true;
+                googletag.cmd.push(function() {
+                    pbjs.que.push(function() {
+                        pbjs.setTargetingForGPTAsync();
+                        googletag.encryptedSignalProviders= [];
+                        // Timeout added for now to get eids from all the partners before registering the signal
+                        setTimeout(function(){
+                            var signalSources =  [  
```

Comment:
I think we should get this value from config, filling #8067 , rather than listing them out here

Reply:
perhaps userSync.encryptedSignalSources ?

Reply:
Do you mean we should set the signalSources object in the config under userSync and then retrieve them within registerSignalSources? I definitely think it makes sense to make it part of the Prebid config.

Reply:
@r-schweitzer encryptedSignalSources Object added in the config under userSync and retrieved in registerSignalSources. Please find the object under userSync in description part